torch/distributed/optim/utils.py

```diff
@@ -24,6 +24,20 @@
     optim.Adamax: _FunctionalAdamax,
 }
 
+
+# Interface to insert a new functional optimizer to functional_optim_map
+# fn_optim_key and fn_optimizer are user defined
```

Comment:
Per https://github.com/pytorch/pytorch/pull/80438#discussion_r920714097, could we maybe write:
```suggestion
# ``fn_optim_key`` and ``fn_optimizer`` are user-defined, where ``fn_optim_key`` may not necessarily
# be a :class:`torch.optim.Optimizer` (e.g. for custom optimizers)
```
(rewrap the line as you like)